fulp_modules/lisa/velvet_fu.dm

```diff
@@ -0,0 +1,292 @@
+		owner.mind.martial_art.streak = "twisted_stance"
+		H.adjustStaminaLoss(-40)
+		H.apply_damage(18, BRUTE, BODY_ZONE_CHEST, wound_bonus = CANT_WOUND)
+
+/datum/martial_art/velvetfu/proc/twisted_stance(mob/living/carbon/user)
+	return // Empty return because we only want the button trigger
+
+
+/// Flying Axe Kick - Deals Brute and causes bleeding. Costs 50 Stamina.
+/datum/martial_art/velvetfu/proc/flyingAxekick(mob/living/A, mob/living/D)
+	log_combat(A, D, "flying kick (Velvet-Fu)")
+	D.visible_message("<span class='danger'>[A] flying kicked [D], such skill!</span>", \
+					"<span class='userdanger'>Your neck is flying kicked by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You flying kick [D]!</span>")
+	A.adjustStaminaLoss(50)
+	if(prob(60))
+		if(!D.stat)
+			var/obj/item/bodypart/limb = D.get_bodypart(ran_zone(A.zone_selected))
+			var/datum/wound/slash/moderate/crit_wound = new
+			crit_wound.apply_wound(limb)
+	else
+		to_chat(A, "<span class='danger'>Your flying axe kick fails to cause [D] to bleed!</span>")
+	D.apply_damage(10, BRUTE) // Slash!
+	playsound(get_turf(A), 'sound/weapons/slice.ogg', 50, TRUE, -1)
+	return TRUE
+
+/// Goat Headbutt - Deals Brute and Stuns, in exchange for causing Brute to the user. Costs 60 Stamina and deals some Brute.
+/datum/martial_art/velvetfu/proc/goatHeadbutt(mob/living/A, mob/living/D)
+	log_combat(A, D, "goat headbutt (Velvet-Fu)")
+	D.visible_message("<span class='danger'>[A] headbutted [D]!</span>", \
+					"<span class='userdanger'>You're headbutted by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You swiftly headbutt [D]!</span>")
+	A.apply_damage(15, BRUTE, BODY_ZONE_HEAD, wound_bonus = CANT_WOUND)
+	A.adjustStaminaLoss(60)
+	if(prob(80))
+		if(!D.stat)
+			D.Stun(4 SECONDS)
+			D.Jitter(6 SECONDS)
+	else
+		to_chat(A, "<span class='danger'>You fail to stun [D]!</span>")
+	D.apply_damage(10, A.get_attack_type(), BODY_ZONE_CHEST, wound_bonus = CANT_WOUND)
+	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, TRUE, -1)
+	return TRUE
+
+/// Full Thrust - Deals Brute and has a chance to knock the opponent down. Costs 70 Stamina.
+/datum/martial_art/velvetfu/proc/fullThrust(mob/living/A, mob/living/D)
+	log_combat(A, D, "full thrust (Velvet-Fu)")
+	D.visible_message("<span class='danger'>[A] thrusted into [D]!</span>", \
+					"<span class='userdanger'>You're thrusted by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You quickly and fashionably thrust into [D]!</span>")
+	A.adjustStaminaLoss(70)
+	if(prob(60))
+		if(!D.stat)
+			D.Knockdown(30)
```

Comment:
why the hell would I ever use this over goat headbutt

Reply:
I... guess you can use this to get extra damage on your harm and disarm attacks? or you could just hardtun someone with goat headbutt and just knock them down while they can't dodge/move them onto some soap or something